scss/util/_color.scss

```diff
@@ -23,7 +23,7 @@
     $rgb: nth($rgba, $i);
     $rgb: $rgb / 255;
 
-    $rgb: if($rgb < 0.03928, $rgb / 12.92, pow(($rgb + 0.055) / 1.055, 2.4));
+    $rgb: if($rgb < 0.03928, $rgb / 12.92, pow(($rgb + 0.055) / 1.055, 2.4, 18));
```

Comment:
From what I can tell the `pow()` function is only used here, so it may be better just to change the default. @ncoden, let me know if you want me to make the change.